packages/rspack/src/rspack.ts

```diff
@@ -64,7 +66,7 @@ export async function bundle (nuxt: Nuxt) {
 
     // In dev, write files in memory FS
     if (nuxt.options.dev) {
-      compiler.outputFileSystem = mfs as unknown as OutputFileSystem
+      compiler.outputFileSystem = mfs as any /* as OutputFileSystem */
```

Comment:
I'm not sure it need at rspack, `getRspackMemoryAssets` should work for dev server visit resource. Maybe it used for online edit?